__tests__/schema/posts/boost.ts

```diff
@@ -105,6 +106,12 @@ beforeEach(async () => {
   jest.clearAllMocks();
   await ioRedisPool.execute((client) => client.flushall());
 
+  // Mock transferCores to succeed by default - will be overridden in specific tests
+  (transferCores as jest.Mock).mockResolvedValue({
+    senderBalance: { newBalance: '10000' },
+    receiverBalance: { newBalance: '5000' },
+  });
```

Comment:
There is a `createMockNjordTransport` inside which you can setup accounts balance through njord calls and the e2e test it without faking anything